doc/_static/css/custom.css

```diff
@@ -99,6 +102,87 @@ div.card-body details[open] summary i {
     display: none;
 }
 
+/** Landing page features *************************************************************/
+
+.hero-title {
+    color: #DA7007; /* Darker text color for the title */
+    font-family: "Poppins";
+    font-size: 2.5rem;
+    margin-bottom: 4rem;
+    margin-top: 0px;
+}
+
+.hero-description {
+    font-family: "Nunito Sans";
+    text-align: left;
+    font-size: 1.15rem;
+    margin-bottom: 10px;
+}
+
+.row-padding-main-container {
+    padding-top: 5rem !important;
+    padding-bottom: 5rem !important;
+}
+
+.row-padding-between-features {
+    padding-bottom: 2rem !important;
+    padding-top: 2rem !important;
+}
+
+.feature-title {
+    color: #DA7007; /* Darker text color for the title */
+    font-family: "Poppins";
+    font-size: 1.8rem;
+    margin-bottom: 1.25rem; /* Spacing below title */
+    margin-top: 0px;
+}
+
+.feature-text {
+    font-family: "Nunito Sans";
+    font-size: 1.05rem; /* Regular font size for text */
+    margin-bottom: 0; /* Remove default bottom margin */
+}
+
+.feature-text-see-also {
+    font-family: "Nunito Sans";
+    font-size: 1.2rem;
+    margin-top: 1.75rem;
+}
+
+.feature-text-see-also a {
+    color: var(--pst-color-text-base);
+    font-weight: 600;
+    text-decoration: none;
+}
+
+.feature-text-see-also a:hover {
+    color: var(--pst-color-text-base);
+    font-weight: normal;
+}
+
+.code-margin {
+    margin-top: 0.75em !important;
+    margin-bottom: 0.75em !important;
+}
+
+html[data-theme="light"] .encoding-image {
+    background-image: url('../encoding_light.png');
```

Comment:
@Vincent-Maladiere Do you have the code that generate the image `_static/gap.png`? I have a placeholder but I would like to generate the original one.

Reply:
Is this resolved? (cf IRL convo)